Qa 75 production gitlab pipeline

mzedel opened the pull request.

lluiscampos commented on .gitlab-ci.yml:
```diff
@@ -0,0 +1,199 @@
+    # Verify that the Swagger docs are valid
+    - swagger-cli validate docs/*.yml
+    # Verify that the Swagger docs follow the autodeployment requirements
+    - python2.7 verify_docs.py `find docs -name "*.yml"`
```
We can change this to Python 3 call once we merge [this PR](https://github.com/mendersoftware/autodocs/pull/1).

lluiscampos replied:
The mentioned PR is now merged, can we please switch to Python 3? Also in `device_auth`, `deviceadm`, `inventory`, `tenantadm`, `mender-cli` and `useradm`.

lluiscampos commented on .gitlab-ci.yml:
```diff
@@ -0,0 +1,199 @@
+    - docker load -i acceptance_testing_image.tar
+    - docker load -i tests_image.tar
+    - TESTS_DIR=${SHARED_PATH}/tests ${SHARED_PATH}/tests/run-test-environment acceptance ${SHARED_PATH}/mender-integration ${SHARED_PATH}/tests/docker-compose-acceptance.yml ;
+    - TESTS_DIR=${SHARED_PATH}/tests ${SHARED_PATH}/tests/run-test-environment acceptance ${SHARED_PATH}/mender-integration ${SHARED_PATH}/tests/docker-compose-acceptance.mt.yml ;
```
My bad understanding here. Looks good

lluiscampos commented on .gitlab-ci.yml:
```diff
@@ -0,0 +1,199 @@
+    - docker load -i tests_image.tar
+    - TESTS_DIR=${SHARED_PATH}/tests ${SHARED_PATH}/tests/run-test-environment acceptance ${SHARED_PATH}/mender-integration ${SHARED_PATH}/tests/docker-compose-acceptance.yml ;
+    - TESTS_DIR=${SHARED_PATH}/tests ${SHARED_PATH}/tests/run-test-environment acceptance ${SHARED_PATH}/mender-integration ${SHARED_PATH}/tests/docker-compose-acceptance.mt.yml ;
+    - mv ${SHARED_PATH}/tests/coverage-acceptance.txt $CI_PROJECT_DIR
```
And here :)

lluiscampos commented on .gitlab-ci.yml:
```diff
@@ -0,0 +1,199 @@
+    - test:prepare_acceptance
+  script:
+    - apk add git bash
+    - 'export SHARED_PATH="$(dirname ${CI_PROJECT_DIR})"'
```
:+1:

mzedel replied:
btw. that was adapted from a Gitlab suggestion and seemed to work, I guess if the whole line is just a literal the literals content will be executed instead 🤷‍♂

mzedel commented:
Good point to clean all this. I changed the ones in question and removed the part from `mender-cli` - turns out it's not used there at all 🎉 and since it's deprecated I won't change `deviceadm`.

lluiscampos commented:
Re-running integ tests just to be in the safe side. See [Jenkins build](https://mender-jenkins.mender.io/job/mender-builder/3877/).
